front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -30,29 +30,23 @@ export default function SpeedLimitByTagSelector({
   );
 
   if (!speedLimitsTagsList.length) return null;
+
   return (
     <div className="osrd-config-item mb-2">
       <div
         className={cx('osrd-config-item-container', {
           'd-flex align-items-center gap-10': condensed,
         })}
       >
-        <img width="32px" src={icon} alt="speedometer" />
-        <span className="text-muted">{t('speedLimitByTag')}</span>
-        <SelectImprovedSNCF
-          sm
+        <Select
```

Comment:
This will also change the UI for the selector in operational studies when adding/editing a train, resulting to a "zombified" interface. Do we have the ok from PO's or @thibautsailly ?
Moreover, in operational studies, the label will dislay "Code compo" instead of "Code de composition". Is it a change we want ?

Reply:
Ok for @thibautsailly